src/scripts/libs/download-screenshot.ts

```diff
@@ -25,8 +25,12 @@ export function downloadScreenshot(
   const ctx = finalCanvas.getContext('2d');
   const esaLogo = new Image();
   const padding = 10;
-  const usageInfo = `${mainLayer?.usageInfo ?? ''} ${compareLayer?.usageInfo ??
-    ''}`;
+  // avoid showing the same usage info twice
+  const compareLayerUsageInfo =
+    compareLayer?.usageInfo && compareLayer.usageInfo !== mainLayer?.usageInfo
+      ? compareLayer.usageInfo
+      : '';
+  const usageInfo = `${mainLayer?.usageInfo ?? ''} ${compareLayerUsageInfo}`;
```

Comment:
A bit confusing, how about? (Not sure if really less confusing or only in my head)
```js
const usageInfos = [mainLayer?.usageInfo, compareLayer?.usageInfo]
const usageInfo = [...new Set(usageInfos)].filter(Boolean).join(' ')
```